corehq/apps/reports/standard/forms/reports.py

```diff
@@ -56,13 +56,24 @@ def submitfilter(self):
             self._submitfilter = SubmissionTypeFilter.get_filter_toggle(self.request)
         return self._submitfilter
 
+    @property
+    def sort_params(self):
+        sort_col_idx = int(self.request.GET.get('iSortCol_0', 2))
+        col = self.headers.header[sort_col_idx]
+        sort_prop = col.prop_name
```

Comment:
could this ever be a column that isn't sortable / doesn't have a `prop_name`?

Reply:
It shouldn't be, since I specifically made each column either sortable or have a prop_name, but i might as well add something defensive